src/main/webapp/app/admin/organization-management/organization-management.service.ts

```diff
@@ -22,8 +22,8 @@ export class OrganizationManagementService {
      * Send GET request to retrieve the number of users and courses of
      * all organizations
      */
-    getNumberOfUsersAndCoursesOfOrganizations(): Observable<any> {
-        return this.http.get(this.resourceUrl + '/count-all');
+    getNumberOfUsersAndCoursesOfOrganizations(): Observable<any[]> {
+        return this.http.get<any[]>(this.resourceUrl + '/count-all');
```

Comment:
Not any but something like {numberOfUsers:number, numberOfCourses:number) might be more helpful for future use.

Reply:
Thanks :+1: 
I've now updated the type with e932daa